context/td-context.jsonld

```diff
@@ -114,7 +114,8 @@
       "@id" : "td:isSynchronous"
     },
     "instance": {
-      "@id": "td:instance"
+      "@id": "td:instance",
+      "@type": "xsd:string"
```

Comment:
> Was there a reason you added this type explicitly ?

I don't recall why, but I think it caused problems with the renderer. Sorry, but where is the issue exactly? I think the json schema already enforces that to be a string, it should never be a number or something else.

Reply:
The problem is that with the json-ld api in its current state, you can not perform roundtripping on a TD if you put a `"@type": "xsd:string"` in the context. 
- JSON-LD to Ntriples [playground](https://json-ld.org/playground/#startTab=tab-nquads&json-ld=%7B%22%40context%22%3A%7B%22xsd%22%3A%22http%3A%2F%2Fwww.w3.org%2F2001%2FXMLSchema%23%22%2C%22instance%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhctl.example.org%2Finstance%22%2C%22%40type%22%3A%22xsd%3Astring%22%7D%7D%2C%22%40id%22%3A%22urn%3A123%22%2C%22instance%22%3A%22something%22%7D&frame=%7B%7D&context=%7B%7D)
- NTriples to JSON-LD give
```
[
  {
    "@id": "urn:123",
    "http://hctl.example.org/instance": [
      {
        "@value": "something"
      }
    ]
  }
]
```
- JSON-LD framing JSON-LD again [playground](https://json-ld.org/playground/#startTab=tab-framed&json-ld=%5B%7B%22%40id%22%3A%22urn%3A123%22%2C%22http%3A%2F%2Fhctl.example.org%2Finstance%22%3A%5B%7B%22%40value%22%3A%22something%22%7D%5D%7D%5D&frame=%7B%22%40context%22%3A%7B%22xsd%22%3A%22http%3A%2F%2Fwww.w3.org%2F2001%2FXMLSchema%23%22%2C%22instance%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhctl.example.org%2Finstance%22%2C%22%40type%22%3A%22xsd%3Astring%22%7D%7D%2C%22%40id%22%3A%22urn%3A123%22%7D&context=%7B%7D) --> `"http://hctl.example.org/instance"` was not framed into `instance` because of the `xsd:string`

- Framing without the `"@type": "xsd:string"`: [playground](https://json-ld.org/playground/#startTab=tab-framed&json-ld=%5B%7B%22%40id%22%3A%22urn%3A123%22%2C%22http%3A%2F%2Fhctl.example.org%2Finstance%22%3A%5B%7B%22%40value%22%3A%22something%22%7D%5D%7D%5D&frame=%7B%22%40context%22%3A%7B%22xsd%22%3A%22http%3A%2F%2Fwww.w3.org%2F2001%2FXMLSchema%23%22%2C%22instance%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhctl.example.org%2Finstance%22%7D%7D%2C%22%40id%22%3A%22urn%3A123%22%7D)

Reply:
Checked again having it without the type causes the generation of this triple:
> _:c14n0 <https://www.w3.org/2019/wot/td#instance> "0.5.1"@en 

I believe it is erroneous to add the language tag to a string that is supposed to be a version code...